README.md

```diff
@@ -127,6 +127,7 @@ traceAPICalls|false| Activates tracing of API calls or not
 transport|axios|Overrides the transport layer
 noStorage|false|De-activate using of local storage
 storage|localStorage|Overrides the local storage for caches
+onSessionExpiration|undefined|Callback to run when the session token is expired
```

Comment:
We need a section in the readme which explain how to use and gives an example. For instance, what is supposed to be implemented in the callback. Should the callback call logon again? Should it return anything?

Reply:
done